<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leeRenameTool</class>
 <widget class="QWidget" name="leeRenameTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>leeRenameTool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/avatar.jpg</normaloff>../../../Downloads/avatar.jpg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="BrowserLayout" stretch="0,0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="browserLabel">
         <property name="text">
          <string>Browser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="browserEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="browserButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="NewNameLayout" stretch="0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="NewnameLabel">
         <property name="text">
          <string>New Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NewnameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="prefixLayout" stretch="0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="prefixLabel">
         <property name="text">
          <string>Prefix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="prefixEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="SuffixLayout" stretch="0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="SuffixLabel">
         <property name="text">
          <string>Subfix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="SuffixEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="ReloadTree">
       <property name="text">
        <string>Reload Tree View</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="FilterLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="FilterLabel">
         <property name="text">
          <string>filter</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FilterEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="FilterImage">
         <property name="text">
          <string>Picture Image</string>
         </property>
         <property name="autoRepeatInterval">
          <number>102</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="FilterFBX">
         <property name="text">
          <string>3D FBX</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="treeView">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>99</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="DoRename">
     <property name="text">
      <string>Do Rename</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
